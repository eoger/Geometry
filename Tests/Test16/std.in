C zc1 1 1 9
R Rect 40 40 -10 -10
LIST
L ligneeee 0 0 1 1
OA Magg ligneeee Rect
PL plpl 1 1 1 2 2 2
LIST
C a 8 8 8
OA 0ui Rect zc1 Magg
LIST
EXIT