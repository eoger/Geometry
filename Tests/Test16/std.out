OK
OK
R Rect 40 40 -10 -10
C zc1 1 1 9
OK
OK
OK
L ligneeee 0 0 1 1
OA Magg Rect ligneeee
PL plpl 1 1 1 2 2 2
R Rect 40 40 -10 -10
C zc1 1 1 9
OK
OK
OA 0ui Magg Rect zc1
C a 8 8 8
L ligneeee 0 0 1 1
OA Magg Rect ligneeee
PL plpl 1 1 1 2 2 2
R Rect 40 40 -10 -10
C zc1 1 1 9
